crates/net/discv5/src/filter.rs

```diff
@@ -40,7 +40,8 @@ impl MustIncludeKey {
         FilterOutcome::Ok
     }
 
-    fn ignore_reason(&self) -> String {
+    /// Returns the message explaining why an [Enr](discv5::Enr) is filtered out.
```

Comment:
are these really useful?

Reply:
can remove - but must say this is out of scope